.github/workflows/docker-build-and-push.yaml

```diff
@@ -0,0 +1,48 @@
+name: Docker Build and Push
+
+on:
+  push:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    env:
+      IMAGE: matejak/argbash
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Set tags
+        id: set_tags
+        shell: bash
+        run: |
+          if [[ "$GITHUB_REF" = refs/tags/* ]]; then
+            version="${GITHUB_REF#refs/tags/}"
+            tags="latest,$version"
+          else
+            branch="${GITHUB_REF#refs/heads/}"
+            branch="branch-${branch/\//-}"
+            tags="$branch"
+          fi
+          echo "::set-output name=tags::$tags"
+          echo "The tags were set to '$tags'"
+          
+      - uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
Is there a reason for it to be secret? I would just put `matejak` in there.

Reply:
No, you're right.